src/content-script.ts

```diff
@@ -1,11 +1,11 @@
 const public_path = (Reflect.get(window, 'browser') || Reflect.get(window, 'chrome')).runtime.getURL('/')
 __webpack_public_path__ = public_path
 const orig = Object.getOwnPropertyDescriptor(HTMLScriptElement, 'src')!
-const chunks = new WeakSet<Node>()
+const __webpack__loading__chunks__ = new Set<Node>()
```

Comment:
we are... taking responsibility know? should we also delete it in a `catch` later or?

Reply:
yeah, webpack's default way doesn't run our code in the content script context. There is a way to not use eval. Let background script do this but that's unnecessary complicated.

I prefer run es module directly but iOS won't work with it. Upstream (realms-shim) block us.

Reply:
We can do it in webpack! See #125.